person.rb

```diff
@@ -28,6 +30,10 @@ def of_age?
   def correct_name
     @name
   end
+
+  def add_rental(date, person)
+    Rental.new(date, self, person)
+  end
```

Comment:
- [ ] Kindly be noticed that you should send the date and the book attributes to the add_rental method in the Person calss.
```ruby
def add_rental(date, book)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
end
```

Reply:
This issue is well resolved! ✔️